sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -23,9 +23,11 @@
 
 #### Features Added
 * Added ability to replace `computed_properties` through `replace_container` method. See [PR 39543](https://github.com/Azure/azure-sdk-for-python/pull/39543)
+* Added ability to use Filters from Logging module on Diagnostics Logging based on Http request/response related attributes. See [PR 39897](https://github.com/Azure/azure-sdk-for-python/pull/39897)
```

Comment:
going to need to move these entries to the relevant newest one - we're probably running a release between today and tomorrow so might want to wait til then